packages/old-import/src/widgets/options.ts

```diff
@@ -77,6 +77,8 @@ const optionMapping: OptionMapping = {
     forecastDayCount: (oldOptions) => oldOptions.forecastDays,
     hasForecast: (oldOptions) => oldOptions.displayWeekly,
     isFormatFahrenheit: (oldOptions) => oldOptions.displayInFahrenheit,
+    disableTemperatureDecimals: (oldOptions) => oldOptions.disableTemperatureDecimals,
```

Comment:
Wait, we already had this in 0.15.10 and before? I didn't know that 😅

Reply:
No we did not, they should not be added to the `old-import` schema above (see packages/old-import/src/widgets/definitions/weather.ts)

Reply:
Ok, so please don't add it @hillaliy